eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -73,7 +75,16 @@ jobs:
         value: ${{ parameters.Location }}
 
     strategy:
-      matrix: ${{ parameters.Matrix }}
+      matrix: 
+        ${{ insert }}: ${{ parameters.Matrix }}
+        ${{ if eq(parameters.TestCanary, 'true') }}:          
+          Linux - Java 8 (AzureCloudCanary):            
```

Comment:
I would probably add a Java 11 run instead of 8.

Reply:
Thanks. Updated as suggested.. let me know if that's good.